README.md

```diff
@@ -582,6 +582,28 @@ A LightOperation is an object containing at least one of a `Light`'s properties,
 ```
 or a subset thereof.
 
+### `Plug`
+A plug represents a single outlet plug and has several properties describing its state:
+* `onOff: boolean` - If the plug is on (`true`) or off (`false`)
+
+as well as a few readonly properties:
+* `isSwitchable: boolean` - Whether the plugs supports on/off (always true).
+* `isDimmable: boolean` - Whether the plug supports setting the dimmer value (always false for now).
+
+If the plug object was returned from a library function and not created by you, the following methods are available to change its appearance directly. You can await them to make sure the commands were sent or just fire-and-forget them. The returned Promises resolve to true if a command was sent, otherwise to false.
```

Comment:
```diff
- appearance
+ state
```
We should probably wrap true and false and so on in backticks, but that applies everywhere, not just here.